test/benchmarks/run_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -ex
+CDIR="$(cd "$(dirname "$0")" ; pwd -P)"
+LOGFILE=/tmp/pytorch_benchmarks_test.log
+VERBOSITY=0
+
+# Note [Keep Going]
+#
+# Set the `CONTINUE_ON_ERROR` flag to `true` to make the CircleCI tests continue on error.
+# This will allow you to see all the failures on your PR, not stopping with the first
+# test failure like the default behavior.
+CONTINUE_ON_ERROR="${CONTINUE_ON_ERROR:-0}"
+if [[ "$CONTINUE_ON_ERROR" == "1" ]]; then
+  set +e
+fi
+
+while getopts 'LV:' OPTION
+do
+  case $OPTION in
+    L)
+      LOGFILE=
+      ;;
+    V)
+      VERBOSITY=$OPTARG
+      ;;
+  esac
+done
+shift $(($OPTIND - 1))
+
+function run_make_tests {
+  MAKE_V=""
+  if [ "$VERBOSITY" != "0" ]; then
+    MAKE_V="V=$VERBOSITY"
+  fi
+  make -C $CDIR $MAKE_V all
```

Comment:
Wondering what the benefits are to use Makefile as opposed to running regular python test, for example "PJRT_DEVICE=CUDA python test/test_train_mp_imagenet_fsdp.py --fake_data --use_nested_fsdp --use_small_fake_sample --num_epochs=1".